compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -1125,6 +1125,11 @@ class Definitions {
     "reactor.util.annotation.NonNullApi" ::
     "io.reactivex.annotations.NonNull" :: Nil)
 
+  @tu lazy val NullableAnnots: List[ClassSymbol] = getClassesIfDefined(
```

Comment:
Can we copy the full list from kotlin and update our existing list `NotNullAnnots` as well: https://github.com/JetBrains/kotlin/blob/master/core/compiler.common.jvm/src/org/jetbrains/kotlin/load/java/JvmAnnotationNames.kt